guide/payments/send.md

```diff
@@ -39,169 +103,255 @@ layout = "full-width"
-   retina = "/assets/images/guide/payments/send/get-recipient-address@2x.png"
-   alt-text = "Graphic showing chat messages between the sender and receiver. The sender sends a bitcoin address and the receiver agrees to pay."
-   width = 400
-   height = 400
-   layout = "float-right-desktop"
-%}
+{% for item in page.imagesEntryPoints %}
 
-To send a payment on the Bitcoin blockchain, the sender needs to obtain an address from the recipient. Since Bitcoin [addresses]({{ '/guide/glossary/address' | relative_url }}) are long and seemingly random, they are best shared by copying and pasting in plain text, as a [payment link]({{ '/guide/designing-products/wallet-interoperability/#payments' | relative_url }}), or as a scannable [QR Code]({{ '/guide/designing-products/wallet-interoperability/#qr-codes' | relative_url }}).
+{% capture imageURL %}{{ page.image_base }}{{ item.file }}.png{% endcapture %}
+{% capture imageURLRetina %}{{ page.image_base }}{{ item.file }}@2x.png{% endcapture %}
+
+{% include picture.html
+   image = imageURL
+   retina = imageURLRetina
+   alt-text = item.alt
+   caption = item.caption
+   width = 250
+   height = 541
+   layout = "shadow"
+%}
+{% endfor %}
 
-The receiver does this by generating a new address in their wallet application, then sharing it with the sender. If the sender and receiver are physically close to each other, scanning the receiver's address as a QR Code will be easy. Still, if they are not, they can send the address as text in any regular communication tool like email, SMS, etc.
 </div>
 
-## Inputting an address
+Since users cannot control how an address or invoice is presented to them, wallets should be highly flexible in terms of input options and interoperability.
 
-<div class="center" markdown="1">
-{% include image.html
-   image = "/assets/images/guide/payments/send/input-address.png"
-   retina = "/assets/images/guide/payments/send/input-address@2x.png"
-   alt-text = "Address input field prompting the sender to paste the address"
-   width = 400
-   height = 219
-   layout = "float-right-desktop"
+Payment information can be shared in many formats and over diverse communication channels. Each has its own advantages and limitations. More details on the [payment request formats]({{ '/guide/payments/send/payment-request-formats/' | relative_url }}) page.
+
+## Presenting payment requests
+
+Once the application has registered the payment request, it should provide clear instructions and options to the user.
+
+<div class="image-slide-gallery">
+
+{% for item in page.imagesEntryScreens %}
+
+{% capture imageURL %}{{ page.image_base }}{{ item.file }}.png{% endcapture %}
+{% capture imageURLRetina %}{{ page.image_base }}{{ item.file }}@2x.png{% endcapture %}
+
+{% include picture.html
+   image = imageURL
+   retina = imageURLRetina
+   alt-text = item.alt
+   caption = item.caption
+   width = 250
+   height = 541
+   layout = "shadow"
 %}
+{% endfor %}
+
+</div>
+
+## Manual payment initiation
```

Comment:
```suggestion
## Manual payment configuration
```

The payment has already been initiated by the requester at this stage, configuration seemed more appropriate.

Reply:
Actually, this is a parallel route in which no payment has been initiated, but the user does this on their own.